pyccel/codegen/printing/ccode.py

```diff
@@ -798,7 +810,7 @@ def get_print_format_and_arg(self, var):
                           ('complex',8) : '(%.12lf + %.12lfj)',
                           ('complex',4) : '(%.12f + %.12fj)',
                           ('int',4)     : '%d',
-                          ('int',8)     : '%ld',
+                          ('int',8)     : LiteralString("%") + CMacro('PRId64'),
```

Comment:
Can you explain why you used the macro `PRId64` instead of a regular `%ld`?
`PRId64` literally expands to `ld`.